### ISSUE TITLE
IPOSTMO DASHBOARD, notification side, number of notifications same as msgs

### ISSUE THREAD

**Issue by stepheljobs** [stepheljobs/blaziken#22]
IPOSTMO DASHBOARD, notification side, number of notifications same as msgs --mira

**Comment by stepheljobs** [comment]
This issue is related with notification functionalities.

**Comment by stepheljobs** [comment]
@anthonyteoxon bro may issue with 'Access-Control-Allow-Origin' with notification server. I also encounter this with uploader server before, what I did is something like this on server.js

app.use(function (req, res, next) {
  res.header('Access-Control-Allow-Origin', config.cors.origin);
  res.header('Access-Control-Allow-Methods', config.cors.methods);
  res.header('Access-Control-Allow-Headers', config.cors.headers);
  res.header('Access-Control-Allow-Credentials', true);
  next();
});

I checked the notification server already on aws, not sure which folder I am going to edit. kindly message me, Will try to fix for ipostmo para di ka din maistorbo. Thankies.

![screen shot 2016-01-23 at 9 32 53 am](https://cloud.githubusercontent.com/assets/8773383/12527249/df39181e-c1b2-11e5-94a2-bfaec1892177.png)
